examples/listener.go

```diff
@@ -30,19 +30,19 @@ func listenerCallback(event EventPeople.Event, context EventPeople.ContextInterf
 var twice = make(chan int)
 
 func RunListener() {
-	EventPeople.ListenTo("resource.custom.*", listenerCallback)
-	EventPeople.ListenTo("resource.origin.*", listenerCallback)
+	EventPeople.GetMessageWithCallback("resource.custom.*", listenerCallback)
```

Comment:
thinking about "GetMessageWithCallback" it's clear method name, it describes what it does, but i don't know if it follows other event_people projects standards

Reply:
Fixed